src/eth_client.rs

```diff
@@ -0,0 +1,146 @@
+use bytes::Bytes;
+use ethereum_types::{Address, U256};
+/// This client is inspired in ethrex EthClient https://github.com/lambdaclass/ethrex/blob/main/crates/l2/utils/eth_client/mod.rs
```

Comment:
Move to top of file and make this a module comment (/// -> //!) ?
Also link gives 404

Reply:
Also could we add a comment as to why we use this low level http client instead of a proper eth client like https://docs.rs/ethers-rs/latest/ethers_rs/struct.Client.html?

Reply:
addressed here https://github.com/lambdaclass/eigenda-client-rs/pull/4